clients/flash/air-client/src/org/bigbluebutton/air/main/views/MainView.as

```diff
@@ -57,9 +62,9 @@ package org.bigbluebutton.air.main.views {
 			_menuButtons.bottom = 0;
 			addElement(_menuButtons);
 			
-			_pollButton = new PollButtons();
-			_pollButton.visible = false;
-			_pollButton.percentWidth = 100;
+			_pollButton = new Button();
+			_pollButton.visible = _pollButton.includeInLayout = false;

```

Comment:
MainView uses BasicLayout and I don't think setting includeInLayout applies with that layout.

Reply:
If `includeInLayout` is set to false, parent component will not include this child size to determine its own size. Here a `BasicLayout` works like a canvas.